src/extras/amd.js

```diff
@@ -54,7 +54,12 @@ import { errMsg } from '../err-msg.js';
         setters: setters,
         execute: function () {
           var amdResult = amdExec.apply(exports, depModules);
-          if (amdResult !== undefined)
+          if (
+            amdResult !== undefined ||
+            (amdResult === undefined &&
+                Object.keys(module.exports).length === 0 &&
+                module.exports.constructor === Object)
```

Comment:
This part seems unnecessary, since the exports object is created within this file and will always have Object as its constructor.